include/seqan3/alphabet/nucleotide/dna5.hpp

```diff
@@ -54,10 +51,14 @@
 namespace seqan3
 {
 
+struct rna5;
```

Comment:
If `dna4` is a class now why is `rna4` a struct still?

Reply:
This is just a forward and it doesn't matter for forwards.